lib/blab.rb

```diff
@@ -0,0 +1,36 @@
+class Blab
+  attr_reader :id, :timestamp, :content, :user_id
+
+  def initialize(id, timestamp, content, user_id)
+    @id = id
+    @timestamp = timestamp
+    @content = content
+    @user_id = user_id
+  end
+
+  def self.all
+    result = DatabaseConnection.query("SELECT * FROM blabs;")
+    result.map do |blab|
+      Blab.new(blab['id'], blab['timestamp'], blab['content'], blab['user_id'])
+    end
+  end
+
+  def self.create(content, user_id)
+    result = DatabaseConnection.query("INSERT INTO blabs (content, user_id)" +
```

Comment:
You have similar SQL (for this and other methods) in your user class. Something a good ORM does is abstract this away. Think about whether you could write a 'DatabaseQueries' class such that here you can call:

```
blabs_data = DatabaseQueries.insert('blabs', { 'content' => content, 'user_id' => user_id })
```
Or, more verbose but arguably better:
```
insert_query = InsertQuery.new('blab');
insert_query.add_field('content', content);
insert_query.add_field('user_id', user_id);
blab_data = insert_query.execute();
```
Why is the second one arguably better? Because as well as abstracting away any need to know SQL syntax the user has a clear and instructive interface. Either version though makes your code more DRY and better maintains single responsibility principle (I would argue that the purpose of this class is to model a Blab, not to build SQL).